File: VB/XtraReport1.vb
Imports System

Namespace WindowsFormsApplication3
    Partial Public Class XtraReport1
        Inherits DevExpress.XtraReports.UI.XtraReport

        Public Sub New()
            InitializeComponent()
            Me.Parameters("paramLandscape").Value = Me.Landscape
        End Sub
        Private Sub XtraReport1_ParametersRequestSubmit(ByVal sender As Object, ByVal e As DevExpress.XtraReports.Parameters.ParametersRequestEventArgs) Handles MyBase.ParametersRequestSubmit
            Me.Landscape = CBool(Me.Parameters("paramLandscape").Value)
        End Sub
    End Class
End Namespace

File: CS/XtraReport1.cs
using System;

namespace WindowsFormsApplication3 {
    public partial class XtraReport1 : DevExpress.XtraReports.UI.XtraReport {
        public XtraReport1() {
            InitializeComponent();
            this.Parameters["paramLandscape"].Value = this.Landscape;
        }
        private void XtraReport1_ParametersRequestSubmit(object sender, DevExpress.XtraReports.Parameters.ParametersRequestEventArgs e) {
            this.Landscape = (bool)this.Parameters["paramLandscape"].Value;
        }
    }
}
